packages/engine/Source/Core/Request.js

```diff
@@ -166,7 +167,7 @@ Request.prototype.clone = function (result) {
   result.serverKey = this.serverKey;
 
   // These get defaulted because the cloned request hasn't been issued
-  result.state = this.RequestState.UNISSUED;
```

Comment:
This was a clear bug uncovered by the new tests. Apparently we were never calling the `Request` constructor with any options.